terraform/gcp/projects/pilot-hubs.tfvars

```
@@ -53,6 +53,24 @@ notebook_nodes = {
     resource_labels : {
       "community" : "neurohackademy"
     },
+  },
+  # Nodepool for OceanHackWeek. Tracking issue: https://github.com/2i2c-org/infrastructure/issues/2879
+  "ohw" : {
+    # We expect around 100 users
+    min : 0,
+    max : 100,
+    machine_type : "n1-highmem-16",
```

```suggestion
    machine_type : "n2-highmem-16",
```

I opened a dedicated issue https://github.com/2i2c-org/infrastructure/issues/2923 suggesting we transition to `n2` from `n1`, if you haven't already applied this I suggest going for `n2` right away - there is quota for both `n1` and `n2` in all GCP projects now.

---

Ah yes, because the last time we had this conversation there was no quota left :D

---

Wiee nice!!!